defaults/test_default.yaml

```diff
@@ -48,3 +48,5 @@ send_email: false
 email_recipients: ['qa@scylladb.com']
 
 collect_logs: false
+
+loader_swap_size: 1024 # Size of swap file for loader node in bytes 1024 * 1MB
```

Comment:
1G should be not enough for the default loader?
Can we increase it to be same as the memory size of default loader instance type?

Reply:
A good idea, we should try to put enough memory, cause we are sure not missing only 1Gb

Reply:
usually the recommended swap is the same as a memry size. Check how muchwe can increase this (but not increasing the root disk size)

Reply:
@bentsi @amoskong @fruch the default loader instance memory is 8GB, but disk size looks 10GB. if Create swap equal to memory size, the 2GB of disk will not be enough, i think

Reply:
# df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        1,8G     0  1,8G   0% /dev
tmpfs           1,9G     0  1,9G   0% /dev/shm
tmpfs           1,9G   17M  1,9G   1% /run
tmpfs           1,9G     0  1,9G   0% /sys/fs/cgroup
/dev/nvme0n1p1   10G  5,9G  4,1G  60% /
tmpfs           374M     0  374M   0% /run/user/1000

# cat /proc/meminfo
MemTotal:        3822536 kB

Reply:
maybe we could enlarge the default diskspace ? we are doing that for the other type of nodes already
Disk isn't being use that much in the loaders, if i'm not mistaken.

```
aws_root_disk_size_monitor: 35  # GB, remove this field if default disk size should be used
aws_root_disk_size_db: 15
```

Reply:
@bentsi what do you say we push this like that, and open a new PR for enlarging the disksize of the loaders, to accommodate bigger swap ?

Reply:
yes, let's keep like this 
@aleksbykov please open a task in Trello to increase the loader disk size

Reply:
Done: https://trello.com/c/A9gkrM0z